python/pyspark/ml/feature.py

```diff
@@ -760,6 +765,60 @@ class StringIndexerModel(JavaModel):
     """
     Model fitted by StringIndexer.
     """
+    @property
+    def labels(self):
+        return self._java_obj.labels
+
+
+class IndexToString(JavaTransformer, HasInputCol, HasOutputCol):
+    """
+    Convert provided indexes back to strings using either the metadata on the input column
```

Comment:
Please copy updated Scala doc here.

Also, please mark as Experimental (as in, e.g., RFormula)